pkg/names/names.go

```diff
@@ -37,10 +37,51 @@ const SERVICE_CA_CONFIGMAP = "openshift-service-ca"
 // that is used in multus admission controller deployment
 const MULTUS_VALIDATING_WEBHOOK = "multus.openshift.io"
 
+// ADDL_TRUST_BUNDLE_CONFIGMAP_NS is the namespace for one or more
+// ConfigMaps that contain user provided trusted CA bundles.
+const ADDL_TRUST_BUNDLE_CONFIGMAP_NS = "openshift-config"
+
+// TRUSTED_CA_BUNDLE_CONFIGMAP_NAME is the name of the ConfigMap
+// containing the combined user/system trust bundle.
+const TRUSTED_CA_BUNDLE_CONFIGMAP_NAME = "trusted-ca-bundle"
+
+// TRUSTED_CA_BUNDLE_CONFIGMAP_NS is the namespace that hosts the
+// ADDL_TRUST_BUNDLE_CONFIGMAP and TRUSTED_CA_BUNDLE_CONFIGMAP_NAME
+// ConfigMaps.
+const TRUSTED_CA_BUNDLE_CONFIGMAP_NS = "openshift-config-managed"
+
+// TRUST_BUNDLE_CONFIGMAP_KEY is the name of the data key containing
+// the PEM encoded trust bundle.
+const TRUST_BUNDLE_CONFIGMAP_KEY = "ca-bundle.crt"
+
+// SYSTEM_TRUST_BUNDLE is the full path to the file containing
+// the system trust bundle.
+//const SYSTEM_TRUST_BUNDLE = "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+// TODO: remove following line used for testing.
+//const SYSTEM_TRUST_BUNDLE = "/usr/local/etc/openssl/cert.pem"
+const SYSTEM_TRUST_BUNDLE = "/Users/daneyonhansen/tmp/test.pem"
+
 // Proxy returns the namespaced name "cluster" in the
 // default namespace.
 func Proxy() types.NamespacedName {
 	return types.NamespacedName{
 		Name: PROXY_CONFIG,
 	}
 }
+
+// TrustedCABundleConfigMap returns the namespaced name of the ConfigMap
+// openshift-config-managed/trusted-ca-bundle trust bundle.
+func TrustedCABundleConfigMap() types.NamespacedName {
+	return types.NamespacedName{
+		Namespace: TRUSTED_CA_BUNDLE_CONFIGMAP_NS,
+		Name:      TRUSTED_CA_BUNDLE_CONFIGMAP_NAME,
+	}
+}
+
+// AddlTrustBundleConfigMapNS returns the namespaced name of the
+// namespace containing the user-provided trust bundle ConfigMap.
+func AddlTrustBundleConfigMapNS() types.NamespacedName {
```

Comment:
seems slightly odd that we're returning a "NamespacedName" type when the value we are returning is just the Namespace?

Reply:
@bparees thanks for catching this. It's an unused func so I'll remove it.